UQ-eLIPSE/crucible-search
# story/VETHUB-153

Add fetch Data function to send query tag to back end
to test:
`yarn test`
to verify:
`yarn dev`
![image](https://github.com/UQ-eLIPSE/crucible-search/assets/93627519/11c2f45a-dbd1-4ac0-839a-86505aeb8505)
